h5netcdf/core.py

```diff
@@ -400,6 +444,10 @@ def _create_dim_scales(self):
                 dims = self._variables[dim].dimensions
                 coord_ids = np.array([dim_order[d] for d in dims], 'int32')
                 h5ds.attrs['_Netcdf4Coordinates'] = coord_ids
+            # Might have to be resized if it already exists.
+            elif h5ds.maxshape == (None, ) and \
```

Comment:
Also, use parentheses to span multiple lines instead of explicit continuations, e.g.,
```
elif (h5ds.maxshape == (None, ) and
      self._current_dim_sizes[dim] != h5ds.shape[0]):
```